src/main/java/com/whitesource/cure/Encoder.java

```diff
@@ -224,4 +224,11 @@ private static boolean isAlphaNumeric(char charToEncode) {
         && (charToEncode < 'A' || charToEncode > 'Z')
         && (charToEncode < 'a' || charToEncode > 'z'));
   }
+
+  private static String formatToString(Object content) {
```

Comment:
add a else if (object instance of String) and else the will throws a run time error